test_conformance/math_brute_force/main.cpp

```diff
@@ -194,6 +194,11 @@ int doTest( const char* name )
                 }
             }
         }
+        else
+        {
+            vlog("Skipping reduced precision testing for device with "
```

Comment:
I've seen discussion recently about having a more standardized way for recording tests that skip, do you happen to know if something happened about that?

Reply:
Yes, support for that will be introduced in #816. I thought about using this framework, however I'm not sure how it would work since the `feature` object would need to skip the test [here](https://github.com/KhronosGroup/OpenCL-CTS/pull/816/files#diff-1c97642f8bb6a546daaa3815c4b8c8e3R693) based on whether it was running in reduced mode or not.